### PR TITLE
Fixed groups not loading & badge limit bug

### PR DESCRIPTION
**Pull request by ahmed-ben10**

### CODE REVIEW

File: packages/server/src/app/user/user.router.js [hunk 1]
```diff
@@ -158,16 +144,3 @@ userRouter.post(
     }
   }
 );
-
-userRouter.get('/:userId/challenges', jwtAuthHeaderValidator(), async (request, response, next) => {
```
Comment (NoNamer777) [inline packages/server/src/app/user/user.router.js:162]: You're sure this route isn't used anymore? I'm getting a usage result when I look for the `/challenges` part of the end-point in the ChallengeService in the web client application?

**Review by NoNamer777 — COMMENTED** [review]
Left a comment for thoughts

File: packages/server/src/app/user/user.router.js (hunk 1 above)
Comment (ahmed-ben10) [reply packages/server/src/app/user/user.router.js:162]: there exist one already on line 99

Comment (NoNamer777) [reply packages/server/src/app/user/user.router.js:162]: Nevermind duplicate code removal is always good 🧹 🧹 🧹

**Review by NoNamer777 — APPROVED** [review]

**Pull request merged**